modules/custom/social_language/social_language.install

```diff
@@ -64,3 +67,20 @@ function social_language_update_12201(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Removing wrong language configuration for original-language from groups.
+ */
+function social_language_update_12301(): void {
+  $group_types = GroupType::loadMultiple();
+  foreach ($group_types as $group_type) {
+    $configuration_name = sprintf('group.type.%s', $group_type->id());
+    $collection_value = sprintf('language.%s', $group_type->language()->getId());
+
+    Database::getConnection()
```

Comment:
Would it maybe be possible to replace the delete query with the following? As it's more specific than a delete query.

    ```
  $config_translation = \Drupal::languageManager()->getLanguageConfigOverride($group_type->language()->getId(), $configuration_name);
      $config_translation->delete();
```